src/components/Feed/Feed.tsx

```diff
@@ -2,24 +2,28 @@ import React from 'react';
 import Card from '../Card/Card';
 import { Article } from '../App/App';
 import './Feed.css';
+import { Link } from 'react-router-dom';
+import './Feed.css';
 
 interface PropsFeed {
   articles: Article[]
 }
 
 const Feed: React.FC<PropsFeed> = ({ articles }) => {
   return (
-    <div className="articles-container">
-      <section className="articles-display">
-        {articles.map(article =>
-          <Card
-          title={article.title}
-          image={article.multimedia[0].url}
-          key={article.title}
-          />
-        )}
-      </section>
-    </div>
+    <Link to="/WellNewsFeed">
```

Comment:
What do you think about changing the URL to '/feed' since it's implied that it's WellNews while in the app.

Reply:
Yea. I suppose it is a bit redundant. I'm on board with that change.